src/components/comment-list-item.component.js

```diff
@@ -201,14 +199,14 @@ class CommentListItemComponent extends Component {
             </View>
           </View>
 
-          {!!commentPresent && (
+          {!!commentPresent &&
             <View style={styles.commentContainer}>
-              <MarkdownHtmlView
-                source={comment.body}
+              <GithubHtmlView
+                source={comment.body_html}
```

Comment:
I think you might have meant to use `comment.body`. Noticing this when I open an issue:

![image](https://user-images.githubusercontent.com/12476932/31049167-74e9d9ba-a5fb-11e7-9f5b-9a28a811ace4.png)

From the looks of it, you may have accidentally swapped it by accident :)

![image](https://user-images.githubusercontent.com/12476932/31049187-ec7bd5aa-a5fb-11e7-945e-7e9c53a0dc28.png)

Reply:
@housseindjirdeh We should be using only comment.body_html from now on instead. I tried the branch again and everything works fine for me. Could you give it a new try?
Fair enough, your comment have made me realize that Adding a comment was broken, I fixed it and completely got rid of comment.body :D

Reply:
Yeah was thinking the same thing, appreciate you diving in and removing `body` entirely since we don't really use it anymore